Description: Enabling the unit tests causes a binary dep on ndesk-dbus to be created. Patch them out.
Forwarded: not-needed
Author: Iain Lane <laney@ubuntu.com>
Index: tomboy.git/configure.in
===================================================================
--- tomboy.git.orig/configure.in	2010-07-04 15:03:35.208602782 +0100
+++ tomboy.git/configure.in	2010-07-04 15:28:43.618602487 +0100
@@ -272,10 +272,10 @@
 #
 # NUnit required for (optional) unit tests
 #
-PKG_CHECK_MODULES(NUNIT, mono-nunit, HAVE_NUNIT="yes", HAVE_NUNIT="no")
+#PKG_CHECK_MODULES(NUNIT, mono-nunit, HAVE_NUNIT="yes", HAVE_NUNIT="no")
 AM_CONDITIONAL(HAVE_NUNIT, test "$HAVE_NUNIT" = "yes")
-AC_SUBST(NUNIT_LIBS)
-AC_PATH_PROG(NUNIT, nunit-console)
+#AC_SUBST(NUNIT_LIBS)
+#AC_PATH_PROG(NUNIT, nunit-console)
 
 #
 # Check for X11 libraries
